src/tools/synth_node.rs

```diff
@@ -35,6 +35,15 @@ impl SyntheticNode {
         Self { inner, receiver }
     }
 
+    /// Starts node bound to localhost and with default [pea2pea::Config]
+    pub async fn start_on_localhost() -> Result<Self, AddrParseError> {
```

Comment:
```suggestion
    pub async fn start() -> Result<Self, AddrParseError> {
```
it will always be started on localhost in these tests, so no need to be explicit here

Reply:
Done